sharding-core/sharding-core-preprocessor/src/test/java/org/apache/shardingsphere/core/preprocessor/segment/select/projection/engine/ProjectionsContextEngineTest.java

```diff
@@ -59,4 +62,40 @@ public void assertProjectionsContextCreatedProperlyWhenSelectItemPresent() {
         ProjectionsContext projectionsContext = new ProjectionsContextEngine(null).createProjectionsContext(null, selectStatement, mock(GroupByContext.class), mock(OrderByContext.class));
         assertNotNull(projectionsContext);
     }
+    
+    @Test
+    public void assertProjectionsContextCreatedProperlyWhenOrderByContextOrderItemsPresent() {
```

Comment:
The test method name should same with to be tested method name, so the better name should be `assert createProjectionsContextWhenOrderByContextOrderItemsPresent`